lib/model/CEventRateModel.cc

```diff
@@ -32,15 +31,13 @@
 #include <model/CResourceMonitor.h>
 #include <model/FrequencyPredicates.h>
 
-#include <boost/iterator/counting_iterator.hpp>
-
 #include <algorithm>
+#include <boost/iterator/counting_iterator.hpp>
+#include <stdint.h>
```

Comment:
We usually split the headers into groups:

- Boost headers
- blank line
- C++ headers
- blank line
- C headers

(We've also been gradually switching from C headers to their C++ equivalents, i.e. `<stdint.h>` to `<cstdint>`, but that will also mean adding `std::` in front of all the sized integer types throughout the file, so if that's a lot of extra changes feel free to leave it to a future PR.)

Reply:
Ah yes, missed that. Those includes were added by CLion doing its thing